test/helper_functions.hpp

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) 2016 Steinwurf ApS
+// All Rights Reserved
+//
+// Distributed under the "BSD License". See the accompanying LICENSE.rst
+
+#include <endian/big_endian.hpp>
+#include <endian/little_endian.hpp>
+#include <endian/endian_stream.hpp>
+
+#include <gtest/gtest.h>
+
+/// Test Helper functions
```

Comment:
I think a high-level description of what each helper function tests would be good. Others reading will be more quickly get into the code